### PR TITLE
N7s aur#182 time service tests submodule

### PR DESCRIPTION
**Pull request by Akoszewski**

### CODE REVIEW

File: requirements/AUR_CBI_MDL-0030/AUR_CBI_MDL-0030.acn [hunk 1]
```diff
@@ -0,0 +1,12 @@
+AUR-CBI-MDL-0030-DATAVIEW DEFINITIONS ::= BEGIN
+
+    --  In this ACN model, you can specify custom binary encoding rules for
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_MDL-0030/AUR_CBI_MDL-0030.acn]: Remove commented out code

File: requirements/AUR_CBI_MDL-0030/work/environment/implem/default/C/src/environment.c [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    !! file. The up-to-date signatures can be found in the .ads file.   !!
+*/
+#include "environment.h"
+#include "logs.h"
+//#include <stdio.h>
+
+
+void environment_startup(void)
+{
+   // Write your initialisation code
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_MDL-0030/work/environment/implem/default/C/src/environment.c]: Remove commented out code

File: requirements/AUR_CBI_MDL-0030/work/timeservice/implem/default/C/Makefile [hunk 3]
```diff
@@ -0,0 +1,18 @@
+VISUAL?=kate
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_MDL-0030/work/timeservice/implem/default/C/Makefile]: Is this needed?

File: requirements/AUR_CBI_MDL-0030/work/system_post.asn [hunk 4]
```diff
@@ -0,0 +1,17 @@
+-- Dataview generated on-the-fly providing information on the system
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_MDL-0030/work/system_post.asn]: IIs this needed?

File: requirements/AUR_CBI_MDL-0030/work/environment/implem/default/environment.pro [hunk 5]
```diff
@@ -0,0 +1,3 @@
+SOURCES += work/environment/C/src/environment.c
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_MDL-0030/work/environment/implem/default/environment.pro]: Is this needed?

File: requirements/AUR_CBI_MDL-0030/work/environment/environment.pro [hunk 6]
```diff
@@ -0,0 +1,4 @@
+SOURCES += work/environment/SDL/src/environment.pr
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_MDL-0030/work/environment/environment.pro]: Is this needed, and even correct?

File: requirements/AUR_CBI_MDL-0030/work/timeservice/implem/default/C/src/timeservice-api.c [hunk 7]
```diff
@@ -0,0 +1 @@
+/home/taste/SpaceCreator/AURORA-Validation/AURORA-Reference-Component-Set/TimeService/work/timeservice/implem/default/C/src/timeservice-api.c
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_MDL-0030/work/timeservice/implem/default/C/src/timeservice-api.c]: Do we need to link to individual files? Would there be a possibility to link to the entire TimeService implementation?

File: requirements/AUR_CBI_MDL-0030/interfaceview.xml [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+    <Property name="Taste::InheritPI" value="true"/>
+    <Property name="Taste::Autonamed" value="true"/>
+</Required_Interface>
+
+    <Implementations>
+        <Implementation name="default" language="C"/>
+    </Implementations>
+</Function>
+
+<Function name="TimeService" is_type="NO" language="C" default_implementation="default" fixed_system_element="NO" required_system_element="NO" FPGA_Configurations="" startup_priority="1">
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_MDL-0030/interfaceview.xml]: Can we include the entire set of TimeService interfaces?

File: requirements/AUR_CBI_MDL-0030/work/timeservice/implem/default/C/src/timeservice-api.c (hunk 7 above)
Comment (Akoszewski) [reply requirements/AUR_CBI_MDL-0030/work/timeservice/implem/default/C/src/timeservice-api.c]: We were trying to link a directory but it appears that (as Rafał said) it is impossible to compile this way so we had to link individual files

File: requirements/AUR_CBI_MDL-0030/work/timeservice/implem/default/C/Makefile (hunk 3 above)
Comment (Akoszewski) [reply requirements/AUR_CBI_MDL-0030/work/timeservice/implem/default/C/Makefile]: No, its generated by "make". Removed.

File: requirements/AUR_CBI_MDL-0030/work/system_post.asn (hunk 4 above)
Comment (Akoszewski) [reply requirements/AUR_CBI_MDL-0030/work/system_post.asn]: No, removed

File: requirements/AUR_CBI_MDL-0030/work/environment/implem/default/C/src/environment.c (hunk 2 above)
Comment (Akoszewski) [reply requirements/AUR_CBI_MDL-0030/work/environment/implem/default/C/src/environment.c]: OK

File: requirements/AUR_CBI_MDL-0030/AUR_CBI_MDL-0030.acn (hunk 1 above)
Comment (Akoszewski) [reply requirements/AUR_CBI_MDL-0030/AUR_CBI_MDL-0030.acn]: OK

File: requirements/AUR_CBI_MDL-0030/work/environment/implem/default/environment.pro (hunk 5 above)
Comment (Akoszewski) [reply requirements/AUR_CBI_MDL-0030/work/environment/implem/default/environment.pro]: No

File: requirements/AUR_CBI_MDL-0030/work/environment/environment.pro (hunk 6 above)
Comment (Akoszewski) [reply requirements/AUR_CBI_MDL-0030/work/environment/environment.pro]: No

File: requirements/AUR_CBI_MDL-0030/interfaceview.xml (hunk 8 above)
Comment (Akoszewski) [reply requirements/AUR_CBI_MDL-0030/interfaceview.xml]: Yes, done

**Review by Lurkerpas — APPROVED** [review]

**Pull request merged**